build.sbt

```diff
@@ -2,26 +2,40 @@ lazy val root = project
   .in(file("."))
   .settings(name := "freestyle-rpc-examples")
   .settings(moduleName := "root")
-  .aggregate(`demo-greeting`, `demo-protocolgen`)
+  .dependsOn(`demo-routeguide`)
+  .aggregate(`demo-routeguide`)
+//  .aggregate(`demo-greeting`, `demo-protocolgen`, `demo-routeguide`)
```

Comment:
The same applies here, the commented code will be studied in https://github.com/frees-io/freestyle-rpc-examples/issues/3